src/components/timer/rangebar.js

```diff
@@ -0,0 +1,19 @@
+import StyledInputRange from './styledInputRange';
+
+export default function RangeBar({value, setValue, name}) {
+	return (
+		<>
+			<StyledInputRange
+				type="range"
+				id={name}
+				name={name}
+				min="00"
+				max="59"
+				value={value}
+				onChange={event => {
+					setValue(Number(event.target.value));
+				}}
+			/>
+		</>
```

Comment:
You don't need the fragments here.
```suggestion
			<StyledInputRange
				type="range"
				id={name}
				name={name}
				min="00"
				max="59"
				value={value}
				onChange={event => {
					setValue(Number(event.target.value));
				}}
			/>
```